server_bashrc.sh

```diff
@@ -15,6 +15,9 @@ function buildninjaic() {
 alias cr="~/kernel-tools/codereview/upload.py --check-clang-format --check-eslint"
 alias activate="source python3-venv/bin/activate"
 
+# gdb setup
```

Comment:
💅 "Ensure the toolchain gdb is used; it's in a separate directory from the other toolchain executables."

Reply:
done